extensions/datasketches/src/main/java/io/druid/query/aggregation/datasketches/theta/SketchMergeAggregatorFactory.java

```diff
@@ -81,7 +85,8 @@ public AggregatorFactory getMergingFactory(AggregatorFactory other) throws Aggre
           name,
           Math.max(size, castedOther.size),
           shouldFinalize,
-          true
+          true,
+          errorBoundsStdDev
```

Comment:
Its not obvious to me that this is correct. Does that mean we should be checking all the other members as well (fieldName, size, shouldFinalize, isInputThetaSketch) to make sure they are equal before merging? If you try to merge two aggregatorFactories that have differing values for errorBoundsStdDev, it doesn't invalidate the merge. It just changes what is returned when you finalize. The estimate will be the same, but you may get different (or no) high and low bounds. That seems like it isn't too much of a concern, where merging two aggregator factories with different field names seems like a much more critical error.